sdks/python/apache_beam/runners/direct/direct_userstate.py

```diff
@@ -225,14 +225,28 @@ def __init__(self, step_context, dofn, key_coder):
 
     self.cached_states = {}
     self.cached_timers = {}
-
-  def get_timer(self, timer_spec, key, window, timestamp, pane):
+    self.cached_timer_families = {}
+
+  def get_timer(
```

Comment:
Would it be better to have separated functions like `get_timer()`, `get_timer_family` instead of using a `boolean`